### PR TITLE
New page: aria-describedby

### PR DESCRIPTION
**Pull request by estelle**
- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by ericwbailey — CHANGES_REQUESTED** [review]
Two big-picture things:

1. I'd like to take a second pass at the examples, more detail in the feedback comment.
2. I know you touch on this lightly, but I'd like to emphasize that `aria-describedby` content should only be a text string, and if there are underlying semantics they might not be announced.

File: files/en-us/web/accessibility/aria/attributes/aria-describedby/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+In the example below, a button that functions as a 'close' button on a dialog is described elsewhere in the document. The `aria-describedby` attribute is used to associate the description with the button.
+
+```html
+<button aria-label="Close" aria-describedby="closeDescription"
+    onclick="myDialog.close()">X</button>
+...
+<p id="closeDescription">Closing this window will discard any information entered and
+return you back to the main page</p>
+```
```
[the remark this answers is not in the document]
Comment (ericwbailey) [reply files/en-us/web/accessibility/aria/attributes/aria-describedby/index.md:57]: Maybe we avoid examples for now?

[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-describedby/index.md]: i edited them and commented them out.

[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-describedby/index.md]: should we just use the w3c example:
```html
<button aria-describedby="trash-desc">Move to trash</button>
...
<p id="trash-desc">Items in the trash will be permanently removed after 30 days.</p>
```

[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-describedby/index.md]: i updated the examples we did have and commented them out... should we uncomment or delete? or go with trash example? or all 3?

[the remark this answers is not in the document]
Comment (ericwbailey) [reply files/en-us/web/accessibility/aria/attributes/aria-describedby/index.md]: I think the trash example would work!

[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-describedby/index.md]: done!

**Review by ericwbailey — APPROVED** [review]

**Pull request merged**